xbstrap/base.py

```diff
@@ -30,6 +30,7 @@
 global_yaml_loader = yaml.SafeLoader
 global_bootstrap_validator = None
 native_yaml_available = False
+changed_source_root = None
```

Comment:
I'd rather let the `Config` constructor (and `config_for_dir()`) take a `override_source_root` argument instead of having this magic global variable.

Reply:
I figured, but didn't do that since the same happens elsewhere and there being over 30 call sites to correct. I'll fix this up tonight.

Reply:
Well, there's no need to support `-S` in the main xbstrap CLI for now, so there would be only one call site to fix.

Reply:
true, true, but I was going for consistency there :P

Reply:
I'll be doing it for all tools anyway, nothing a few vim macros can't fix :)